src/components/EditableLabel/EditableLabel.module.scss

```diff
@@ -79,6 +80,12 @@
 
   &--focus {
     transform: scale(1, 3);
+    height: 1px;
+  }
+
+  &--hover {
+    transform: scale(1, 3);
```

Comment:
czo za poteżne powiększenie xd

Reply:
to jest animacia hovera co się rozciąga wzdłuż lebela